### PR TITLE
test: change oss endpoint

### PR DESCRIPTION
**Pull request by rudyfly**
### Ⅰ. Describe what this PR did
change oss endpoint from beijing to hangkong

### Ⅴ. Special notes for reviews

Signed-off-by: Rudy Zhang <rudyflyzhang@gmail.com>

### CODE REVIEW

**Comment by HusterWan** [comment]
LGTM

File: hack/install/config.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-export OSS_ENDPOINT="oss-cn-beijing.aliyuncs.com"
-export OSS_BUCKET="pouch-oss"
+export OSS_ENDPOINT="oss-cn-hongkong.aliyuncs.com"
+export OSS_BUCKET="pouch-oss-hk"
```
Comment (fuweid) [inline hack/install/config.sh]: how to add this sign?

Comment (rudyfly) [reply hack/install/config.sh]: I don't know. 🐶

Comment (fuweid) [reply hack/install/config.sh]: I like it but I don't know to do that. LoL

**Comment by rudyfly** [comment]
PTAL @fuweid

**Pull request merged**